### ISSUE TITLE
How to calculate validation loss when training?

### ISSUE THREAD

**Issue by ChunJen** [matyasbohacek/spoter#11]
how to add the correct formula code in `utils.py` to calculate val_loss?

![image](https://user-images.githubusercontent.com/46671547/219992074-149eee64-6f02-41da-8e51-636bbb4ed5a9.png)

**Comment by matyasbohacek** [comment]
Thanks for your interest in our work!

The validation loss is already calculated at every epoch, see [train.py, line 180](https://github.com/matyasbohacek/spoter/blob/main/train.py#L180).

If this does not resolve your inquiry, please feel free to re-open the thread.
